OpenRA.Game/Map/Map.cs

```diff
@@ -217,12 +216,18 @@ static object LoadVideos(MiniYaml y)
 
 		[FieldLoader.Ignore] public Lazy<Dictionary<string, ActorReference>> Actors;
 
-		public int PlayerCount { get { return Players.Count(p => p.Value.Playable); } }
+		public int PlayerCount
```

Comment:
This may be better as a `Lazy<int>` (or perhaps as a wrapper to a private lazy property to avoid exposing the lazyness), so that this work only needs to be done once.